runtime/build.gradle.kts

```diff
@@ -17,6 +17,38 @@ kotlin {
         publishAllLibraryVariants()
     }
 
+    jvm()
+
+    js {
+        browser {}
+        nodejs {}
+    }
+
+    // Native targets, according to https://kotlinlang.org/docs/native-target-support.html
+    // Tier 1
+    macosX64()
+    macosArm64()
+    iosSimulatorArm64()
+    iosX64()
+    // Tier 2
+    linuxX64()
+    linuxArm64()
+    // watchosSimulatorArm64()
+    // watchosX64()
+    // watchosArm32()
+    // watchosArm64()
+    tvosSimulatorArm64()
+    tvosX64()
+    tvosArm64()
+    iosArm64()
+    // Tier 3
+    // androidNativeArm32()
+    // androidNativeArm64()
+    // androidNativeX86()
+    // androidNativeX64()
+    mingwX64()
```

Comment:
Will the additional targets (like linux and mingw) add to build times?

Reply:
Short answer: no.

It takes longer to build/publish _this_ library, but does not affect build times of downstream consumers. That is, downstream building for e.g. iOS only deals with the iOS library variant.